package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "build": "npm run build:v2 && npm run build:v3",
     "build:v2": "npm run lint && npm run compile",
-    "build:v3": "neon build --release",
+    "build:v3": "npm run lint && neon build --release",
```

Comment:
Hmm. I'm not sure this is the right place for it either, as `neon build` doesn't use anything that would be linted.

I think on master probably the `compile` script should move into the `dist` script (and be removed), and lint is then a `predist` script.

We're really due an overhaul of all these scripts, there are many more than is necessary.